src/commands/open/index.js

```diff
@@ -1,41 +1,5 @@
-const { flags: flagsLib } = require('@oclif/command')
+const { createOpenCommand } = require('./open')
 
-const { isEmptyCommand } = require('../../utils/check-command-inputs')
-const Command = require('../../utils/command')
-const showHelp = require('../../utils/show-help')
-
-const OpenAdminCommand = require('./admin')
-const OpenSiteCommand = require('./site')
-
-class OpenCommand extends Command {
-  async run() {
-    const { args, flags } = this.parse(OpenCommand)
-
-    // Show help on empty sub command
-    if (isEmptyCommand(flags, args)) {
-      showHelp(this.id)
-    }
-
-    if (flags.site) {
-      await OpenSiteCommand.run()
-    }
-    // Default open netlify admin
```

Comment:
There is behavior change in this command. `ntl open` should show the help and then open the admin page by default.

With this PR, `ntl open` only shows the help. Can we keep the old behavior?

Reply:
I did not notice but already fixed :)